### DISCUSSION TITLE
F4ll High round & Records

### DISCUSSION THREAD

**Discussion by notrickzdumbo** [nzp-team/nzportable#206] [Records and Speedruns]
Post/Discuss your F4ll high rounds!
I'm gonna start with mine which broke the wr. I just trained at the outside part next to power and used the waff and gibs o matic. Unfortunately I encountered a glitch at round 45 which made me not able to sprint or buy stuff but I still managed to push it to 51. Good luck beating this!
![fte-20230219221430-0](https://user-images.githubusercontent.com/98246579/219998698-7faffad9-2b4f-454d-ad4b-a0c151f33711.png)

**Comment by turbiG** [comment]
If i didn't down like a dumbass i could've reached round 70
https://www.youtube.com/playlist?list=PLvJXktGru-NeJLz0Ax9X5gXuXjEa1xk_d
![image](https://github.com/nzp-team/nzportable/assets/143304024/13c209b6-b44e-49dc-b9df-3d7b93b3d41f)

**Reply by blubs** [reply]
woah...

**Reply by mmkkiinn** [reply]
Excellent 👍

**Comment by BCDeshiG** [comment]
My first proper run of 2024, mainly used the Headcracker (upgraded Scoped Kar) and trained around the Browning wall-buy. The game definitely feels way harder than it used to, to the point where I used all my revives (I never used to usually) but the improved spawining means things flow much faster as well since I'm not having to worry about zombies getting stuck.

![fte-20240110234554-0](https://github.com/nzp-team/nzportable/assets/29164668/74b1ab24-a159-4545-98a4-960c13e95b62)

I can't remember what my highest was beforehand, but this is definitely one of my better runs. That being said, I doubt I could compete with a lot of the people here.

**Comment by Kingjaxgaming420** [comment]
![image](https://github.com/user-attachments/assets/1e6e3ffc-3399-41a9-aaa0-6a0d7e29e734)
random freeze on 46 (new pb at least)

**Reply by Kingjaxgaming420** [reply]
![Screenshot (5)](https://github.com/user-attachments/assets/97469f70-29cc-458a-b9e1-aea0bc570472)
another freeze on 53

**Comment by mmkkiinn** [comment]
I am trying 😞![Screenshot_2024-10-28-18-41-51-48_2f85358b2198d26f8aca533d68bee793](https://github.com/user-attachments/assets/27548bfe-b767-4a30-a48f-1bb263e946a9)

**Reply by mmkkiinn** [reply]
Finally i got a new round 
![Screenshot_2024-10-29-14-13-51-41_2f85358b2198d26f8aca533d68bee793](https://github.com/user-attachments/assets/eed933f7-0ed8-45d0-a7f4-f8bfa930fae6)
